### ISSUE TITLE
there was a problem looking for IP

### ISSUE THREAD

**Issue by dounai2333** [Ochii/cso2-master-server#35]
![t lxh o831p 2s 2uxris6](https://user-images.githubusercontent.com/45044404/52207915-de96db00-28b9-11e9-9ec0-392e4ea573bc.png)

left: normal, right: P2P tool modified
if I start server normally, it will works
but when I use some P2P online tool, it changes my local IP to 192.168.2.2, and the server has gone, didn't start normally
I tried to modify interfacepicker.js and try to find out what was wrong, but I didn't find it
instead of create a new virtual network adapter, the P2P tool directly modifies your local network adapter

if you have time, I hope you can fix it soon. my operator has limited my P2P hole punching behavior, so I can only use this kind of software, thanks in advance!

**Comment by Ochii** [comment]
Have you tried starting the server with ```node dist/server.js --ip-address 192.168.2.2```?

**Comment by GEEKiDoS** [comment]
> Have you tried starting the server with `node dist/server.js --ip-address 192.168.2.2`?

He mean he is running under some lan gaming platform like Hamachi or Tunngle but without a tun/tap device like 游侠对战平台(http://pk.ali213.net/).

**Comment by Ochii** [comment]
@dounai2333 which software are you using?

**Comment by dounai2333** [comment]
> @dounai2333 which software are you using?

I'm using the one that GEEKiDoS said, it's an all-chinese software, and yet I haven't tried the command you said, because this is the first time I use nodeJS. I have seen -i / --ip-address, but I don't know how to use it (I actually tried to combine it like "npm run start -i 192.168.2.2" and "nmp run start server -i 192.168.2.2", but it doesn't work hahahaha)

I tried the command you said, and it works. thank you!
